src/imp/openssl.rs

```diff
@@ -202,6 +202,9 @@ impl TlsConnector {
 
 /// OpenSSL-specific extensions to `TlsConnectorBuilder`.
 pub trait TlsConnectorBuilderExt {
+    /// Initialize `TlsAcceptorBuilderExt` from an `SslAcceptorBuilder`.
+    fn create(SslConnectorBuilder) -> Self;
```

Comment:
yes, I'll make the change. Also I wanted to ask if you had a preference of `create` vs `new` for the function name.